### ISSUE TITLE
Mentions-ish?

### ISSUE THREAD

**Issue by ian-battaglia** [Rotonde/beaker#65]
I swear I don't want to add in notifications. Buuuuut– If we're going to have ways for other users to mention you, we should have a way for users to find the post mentioning them, for instance if someone mentions you while you're offline.

**Comment by ian-battaglia** [comment]
I wonder what the most elegant way of instituting this into the existing UI is, without adding a new element. Maybe adding a <hr> between your AVI and name?

**Comment by neauoire** [comment]
Have you looked at the filter command? You can quickly locate your mentions, or someone else', with `filter @name`.

**Comment by ian-battaglia** [comment]
I honestly forgot that was an option. Yeah, that probably serves the purpose. It still might be nice to have a button or function to autofill the bar with that info for dummies like me.

**Comment by ruby0x1** [comment]
I thought clicking the name but that's edit:name, and clicking your handle in an entry was better used for something else (see below), I haven't had an idea for mentions yet

Not for mentions but one other thing that's related is viewing your entries or entries of a user.  My thought was clicking on YOUR handle on YOUR page filters to only your entries. This could be true for all pages. 

View my page -> click my name on an entry = my entries. 
View neauoire -> click @neauoire = only devine's entries.
You could also maybe make it so clicking toggles modes.

mode 0 = all entries
mode 1 = entries
mode 2 = entries without mention/target

**Comment by neauoire** [comment]
Good idea! I made it so if the username is the default username, it suggests an edit, otherwise autocompletes for filter.
Implemented in [client](https://github.com/Rotonde/client)

**Comment by ruby0x1** [comment]
sounds good to me
